pom.xml

```diff
@@ -133,8 +153,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>7</source>
-                    <target>7</target>
+                    <source>8</source>
+                    <target>8</target>
```

Comment:
@mseaton is that ok? I needed this for the `Validator` actually, that is compiled against Java 8.

Reply:
@mks-d yes, this should be no problem.  I'll try to look over / investigate the rest of the PR soon.